src/lighteval/logging/evaluation_tracker.py

```diff
@@ -200,12 +200,19 @@ def save(self) -> None:
                 results=self.metrics_logger.metric_aggregated, details=self.details_logger.compiled_details
             )
 
-    def save_results(self, date_id: str, results_dict: dict):
-        output_dir_results = Path(self.output_dir) / "results" / self.general_config_logger.model_name
-        self.fs.mkdirs(output_dir_results, exist_ok=True)
-        output_results_file = output_dir_results / f"results_{date_id}.json"
-        logger.info(f"Saving results to {output_results_file}")
-        with self.fs.open(output_results_file, "w") as f:
+    def save_results(
+        self, date_id: str | None = None, results_dict: dict | None = None, output_path: str | None = None
+    ):
+        if output_path:
+            fs, output_path = url_to_fs(output_path)
```

Comment:
so it will be overiding output_dir, this isn't very clear. Users should be able to set the output_path in a coherent way with results and details.

Reply:
I agree. Then I will make `output_dir` optional:
- If passed, current behavior.
- If None, then needs an output_path for all `save_` methods.

Reply:
Or simpler: one could use `output_dir` as the base dir for the passed `output_path` (considered this as relative to `output_dir`.

Reply:
I have made `output_dir` optional.